cleanrl_utils/huggingface.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import sys
+from pathlib import Path
+from pprint import pformat
+from typing import List
+
+import numpy as np
+
+HUGGINGFACE_VIDEO_PREVIEW_FILE_NAME = "replay.mp4"
+HUGGINGFACE_README_FILE_NAME = "README.md"
+
+
+def push_to_hub(
+    args: argparse.Namespace,
+    episodic_returns: List,
+    repo_id: str,
+    algo_name: str,
+    folder_path: str,
+    video_folder_path: str = "",
+    revision: str = "main",
+    create_pr: bool = False,
+    private: bool = False,
+):
+    # Step 1: lazy import and create / read a huggingface repo
+    from huggingface_hub import CommitOperationAdd, CommitOperationDelete, HfApi
+    from huggingface_hub.repocard import metadata_eval_result, metadata_save
+
+    api = HfApi()
```

Comment:
```suggestion
    api = HfApi(library_name="cleanrl", library_version="1.0.0")
```

This is optional but helps us understand usage of `huggingface_hub` (e.g. "how many users from 'cleanrl' ?")

Reply:
This gives me 
```
Traceback (most recent call last):
  File "/home/costa/Documents/go/src/cleanrl/cleanrl/dqn_jax.py", line 263, in <module>
    push_to_hub(args, episodic_returns, repo_id, "DQN", f"runs/{run_name}", f"videos/{run_name}-eval")
  File "/home/costa/Documents/go/src/cleanrl/cleanrl_utils/huggingface.py", line 29, in push_to_hub
    api = HfApi(library_name="cleanrl", library_version="1.0.0")
TypeError: __init__() got an unexpected keyword argument 'library_name'
```

Reply:
Oops sorry. Available in next version as well so I'll include it in the follow-up PR :)

Reply:
Sounds good. Oh also a related problem — I noticed the replay.mp4 is not longer displayed at the model card https://huggingface.co/cleanrl/BreakoutNoFrameskip-v4-dqn_atari_jax-seed1. Would you mind checking if that's the case?

Reply:
Just for context, was it ever included in the `README.md` or do you mean the widget displayed on the right side of the model card (above inference API) ? Maybe @simoninithomas could have an insight on this.

Reply:
All good. It's just the other day the video in this screenshot is not showing, but it's working today.
<img width="1444" alt="Screenshot 2022-12-20 at 9 36 53 AM" src="https://user-images.githubusercontent.com/5555347/208692170-31a7fac9-4584-45d9-a554-8daad34ef5e2.png">